router/src/lib.rs

```diff
@@ -1072,6 +1082,16 @@ pub(crate) struct GenerateRequest {
     pub inputs: String,
     #[serde(default = "default_parameters")]
     pub parameters: GenerateParameters,
+
+    /// This is used internally because some requests
+    /// already contain the templated input therefore
+    /// we shouldn't add the special tokens.
+    #[serde(default = "default_true", skip)]
+    pub add_special_tokens: bool,
```

Comment:
Shouldn't we add this to the internal `tokenize`, `infer`... router functions instead?

Reply:
This was way faster to do it this way (as we're passing GenerateRequest everywhere).
Using `[serde(skip)]` means it's not settable through public API nor is it visible with utoipa.

My first attempt I used a separate variable and found it was just adding lots of noise for nothing (we *need* that information on *every* generateRequest).